wca-regulations.md

```diff
@@ -131,7 +131,11 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 3h2) Puzzles whose colored plastic is visible inside the puzzle (e.g. "stickerless" puzzles) are permitted. This does not include the following puzzles:
         - 3h2a) Puzzles with transparent parts that can reveal more information about the state of the puzzle. Exception: An overlay sticker or a logo (see [Regulation 3l](regulations:regulation:3l)).
     - 3h3) Any modifications to a puzzle that result in poor performance by a competitor are not grounds for additional attempts.
-    - 3h4) For Clock, custom "inserts" (the same shape and size as the traditional paper inserts) are permitted, at the discretion of the WCA Delegate. The inserts must have a clear indication of 12 o'clock that matches the original inserts.
+    - 3h4) For Clock, the following modifications are permitted:
+        - 3h4a) Customs "inserts" (the same shape and size as the traditional paper inserts) are permitted, at the discretion of the WCA Delegate. The inserts must have a clear indication of 12 o'clock that matches the original inserts.
+        - 3h4b) Puzzle design customizations that do not give an unfair advantage may be permitted, at the discretion of the WCA Delegate (e.g. checkerboard pattern clocks, faces with a custom design, tape on the side of the clock).
+        - 3h4c) Customizations in which pins may be distinguishable from any other pin of the same side are permitted under the discretion of the WCA Delegate.
+        - 3h4d) Logos anywhere on the puzzle, as long as the inner clock faces remain uncovered.
```

Comment:
```suggestion
        - 3h4d) Logos anywhere on the puzzle, as long as the inner clock faces remain uncovered.
    - 3h5) Customizations that leave the pins loose (i.e. pins that can toggle or recede using gravity instead of being directly pressed) are not considered reasonable wear, and these puzzles must not be permitted.
```

Reply:
this suggested change will cause an error but I will fix this later